Classes/Helper/Mods.php

```diff
@@ -14,8 +14,25 @@
  * The TYPO3 project - inspiring people to share!
  */
 
+
 class Mods
```

Comment:
The mods helper and also the slub helper are used to obtain and / or change certain document data, e.g. for display purposes in the list module or to add individual data when publishing, e.g. add a URN or set a date. We definitley should find a better place for this functionality. I suggest doing this at a later time together with the installation of the new internal format, because then mods and slub will have to be changed anyway.